src/components/entries/FEEL/Feel.js

```diff
@@ -179,6 +186,14 @@ function FeelTextfieldComponent(props) {
     setLocalValue(value);
   }, [ value ]);
 
+  // close popup on unmount
```

Comment:
:spiral_notepad: Generic unmount events are fired here to replace the popup closing through the context we used to do.